### ISSUE TITLE
Temporal visualization: misleading spikes due to 1-year overlaps 

### ISSUE THREAD

**Issue by atomrab** [periodo/periodo-client#133]
I just realized, in the context of building a small collection for a project focused on Ethiopia, that the time-selection widget registers two definitions that share a single year as a spike in the display (so if you have one period that ends in 1434 and the next that begins in 1434, the count suddenly jumps to 2 for a single year and then falls back down). This is hard to parse visually in individual collections (it more or less disappears in the aggregate). It's likely to happen in a lot of collections. If we're keeping a visualization like this, can we set it to ignore overlaps of only one year in the display? Or is that something that it's important to preserve in some other context?

**Comment by ptgolden** [comment]
This was irking me today too, but I'm not sure how to get around it.

This is due to the code here: https://github.com/periodo/periodo-client/blob/0099df4af5309281a0a60ca0ab4bf0f2c5c0b16b/modules/periodo-app/src/layouts/authorities/Timeline/Histogram.js. This timeline style works by separating out the timeline into a series of "bins" which have a minimum and maximum year, putting each period into each bin that it falls in, and then putting a series of dots to show the relative amount of periods in each bin. It's not actually due to 1-year overlaps per se, it's due to consecutive periods being placed in the same bin.

Say we have three periods:
Early Babbo: 1000-1420
Middle Babbo: 1420-1840
Late Babbo: 1840-1914

And we need to split that into different columns to represent the span as a histogram. Let's say 10 bins, representing hundred year chunks. Here's where the periods would fall (using E, M, L)

1000-1100 E
1100-1200 E
1200-1300 E
1300-1400 E
1400-1500 EM
1500-1600 M
1600-1700 M
1700-1800 M
1800-1900 ML
1900-2000 L

There's that characteristic weird "bump"-- it's from periods having overlaps that aren't exactly the same as the boundaries of the bin. A naive way of getting around this would be to say: if there are two periods where period A's latest stop is exactly the same as period B's earliest start, then don't put period B in a bin.

**Comment by rybesh** [comment]
I wonder if we could make this more easily understandable by labeling the bins more clearly somehow. It makes total sense in your example above, but is less understandable in the visualization (probably because we want to read it as a timeline rather than as a histogram).

**Comment by ptgolden** [comment]
How about this?

![image](https://user-images.githubusercontent.com/376031/85325122-17c4f080-b499-11ea-9459-ee00d77c6568.png)

**Comment by ptgolden** [comment]
Different algorithm for doing a histogram without discrete bin intervals implemented in f7fccf2495d6074e9b8e0db1f22db63204aff36f

**Comment by atomrab** [comment]
I really like that representation. Will it work with the highlighting that the dev version of the client has now? I also like the highlighting and wouldn't want to lose it.

**Comment by ptgolden** [comment]
Yes! I made sure it stayed in.
